dali/pipeline/operators/reader/loader/video_loader.cc

```diff
@@ -184,8 +185,11 @@ OpenFile& VideoLoader::get_or_open_file(const std::string &filename) {
   if (!file.open) {
     LOG_LINE << "Opening file " << filename << std::endl;
 
+    errno = 0;
     AVFormatContext* raw_fmt_ctx = nullptr;
-    if (avformat_open_input(&raw_fmt_ctx, filename.c_str(), NULL, NULL) < 0) {
+    int ret = avformat_open_input(&raw_fmt_ctx, filename.c_str(), NULL, NULL);
+    if (ret < 0) {
+      if (errno) DALI_WARN(strerror(errno));
       DALI_FAIL(std::string("Could not open file ") + filename);
```

Comment:
If avformat_open_input() fails not because of syscall failure but some other reason, like maybe container not supported and errno is still zero.
Then the error will be printed as `"Could not open file filename.mp4 Success"`

Reply:
It is inside `if (ret < 0) {` so in that case `avformat_open_input` hasn't returned 0 which is success but some error code we should print.